utils/uploadFile.ts

```diff
@@ -0,0 +1,52 @@
+export interface UploadFileData {
+  epubMetadata?: {
+    epubFileName: string
+    title: string
+    author: string
+    language: string
+    coverData: string
+    description: string
+    tags: string[]
+    thumbnailArweaveId: string
+    thumbnailIpfsHash: string
+  }
+  fileRecords: Array<{
+    fileName: string
+    fileType: string
+    arweaveId: string
+    arweaveLink: string
+    ipfsHash: string
+    arweaveKey?: string
+  }>
+}
+
+export function setUploadFileData (data: Partial<UploadFileData>) {
+  try {
+    sessionStorage.setItem('uploadFileData', JSON.stringify(data))
```

Comment:
Refactor `uploadFileData` to const, and consider renaming to something more verbose, e.g. `publish_book_uploaded_file_data`

Reply:
[🎨 Rename sessionStorage key](https://github.com/likecoin/nft-book-press/pull/338/commits/66c56489cfe44b8df1cfa3d2be842554e64e23a5)